topics/materials-science/tutorials/muspinsim/tutorial.md

```diff
@@ -0,0 +1,930 @@
+achieved by providing a vector rather than scalar here).
+
+We have also defined the orientations needed to do our powder average. It is
+possible to define a specific range of angles for the average (see
+[the main MuSpinSim documentation](https://muon-spectroscopy-computational-project.github.io/muspinsim/input/#orientation))
+but there are also helper functions that cover a full sphere. We are using
+`eulrange`, which will generate a grid of $$n \times n \times n$$ (1000)
+weighted Euler angles.
+
+## MuSpinSim Simulate
+
+As all our changes are captured in the config file, this step operates exactly
+as before.
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Simulate](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim/muspinsim/1.1.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Configuration file containing simulation parameters"*: `muspinsim input file hfine_powder` (Input dataset)
+> 2. Should generate the following output(s):
+>    - {% icon param-collection %} `Muspinsim output collection` (Output dataset)
+>    - {% icon param-file %} `muspinsim log for muspinsim input file hfine_powder` (Output dataset)
+>
+{: .hands_on}
+
+## MuSpinSim Plot
+
+Similarly aside from labelling there are no differences with the plotting
+parameters.
+
+> <hands-on-title>Tool details</hands-on-title>
+>
+> 1. {% tool [MuSpinSim Plot](toolshed.g2.bx.psu.edu/repos/muon-spectroscopy-computational-project/muspinsim_plot/muspinsim_plot/3.5.1+galaxy0) %} with the following parameters:
+>    - *"Plot Title"*: `Hyperfine (with powder averaging) Example`
+>    - *"Label for x axis"*: `t`
+>    - *"Label for y axis"*: `Asymmetry`
+>    - *"Output File Type"*: `PNG`
+>    - *"Muspinsim File Series"*:
+>      - *"Muspinsim File Series 1"*:
+>        - {% icon param-file %} *"Muspinsim Experiment Data (.dat)"*: `hfine_powder.dat`
+>        - *"Line/Point Color"*: `Black`
+>        - *"Series Type"*: `Line`
+>        - *"Line type"*: `Solid`
+>        - *"Line width"*: `1.0`
+> 2. Should generate the following output(s):
+>    - {% icon param-file %} `MuSpinSim Plot on data 4` (Output dataset)
+>
+{: .hands_on}
+
+![Resultant plot of asymmetry against time for the powdered example](../../images/powder_example.png "The asymmetry oscillates, but also decays over time")
+
+As discussed earlier, our final plot is the average of 1000 individual
+oscillations. Each has its own frequency, and so as time progresses they end up
+increasingly out of phase. The result is that the oscillations decay as the
+contributions begin to cancel each other out.
+
+# Workflow 4 - Avoided Level Crossing
+
+This workflow simulates a simple {ALC} experiment which involves three spins: a
+muon, an electron, and a hydrogen atom. Both muon and hydrogen are coupled via
+hyperfine coupling to the electron, and so we will need to define a spin
+interaction for each in our configuration file.
```

Comment:
Still doesn't quite make sense to me, if its changed here I will update the docs as well in my latest pr.
```suggestion
muon, an electron, and a hydrogen atom. Both muon and hydrogen are coupled via
the hyperfine coupling to the electron, and so we will need to define a spin
interaction for each in our configuration file.
```

Reply:
I've reworded it again, added that `the` but also re-ordered the sentence and replaced `hyperfine coupling` with `hyperfine interaction` to avoid overusing the term `coupling`. Also added some further detail on the end to hopefully be more explicit.